locale/es/manageiq.po

```diff
@@ -12902,7 +12902,7 @@ msgstr ""
 #: ../app/helpers/service_helper/textual_summary.rb:281
 #: ../config/yaml_strings.rb:1519
 msgid "Cloud"
-msgstr "Nube"
+msgstr "Cloud"
```

Comment:
Is this right?  it is nube elsewhere, e.g. "Base de datos en la nube" for "Cloud Databases"

Reply:
es translator update this string in ICT pass 2, maybe he thought it is no need to be translated here. I think it is OK and he will verify the translation in final pass.

Reply:
Let's verify this change is fine, then we can merge, generate the ui class json files and backport them to get in a build.

Reply:
Hi @jrafanie @Fryguy can we merge this pr and generated a build, then we can let es translator verify this string on the UI ?

Reply:
Hi @Fryguy @jrafanie , I open a ticket to Straker, and they reply that "I'll ask Straker es translator to redo the change to 'Nube' . This makes sense!" Thank you.